cmd/redis-ts-adapter/main.go

```diff
@@ -248,15 +209,15 @@ func serve(addr string, writers []writer, readers []reader) error {
 }
 
 func main() {
-	writers, readers := buildClients(cfg)
+	client := buildClient(cfg)
 	log.WithFields(log.Fields{"address": cfg.listenAddr}).Info("listening...")
-	if err := serve(cfg.listenAddr, writers, readers); err != nil {
+	if err := serve(cfg.listenAddr, client, client); err != nil {
```

Comment:
I guess we can serve with a single client, and just call it client, instead of reader and writer, no?